.github/workflows/static.check.scripts/newline.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+##
+# Imported from TAOS-CI
+# Copyright (c) 2018 Samsung Electronics Co., Ltd. All Rights Reserved.
+# Modified for Github-Action in 2024.
+#
+# Argument 1 ($1): the file containing the list of files to be checked
+
+##
+# @file newline.sh
+# @brief Check if there is a newline issue in a text file. Originally pr-prebuild-newline.sh
+# @see      https://github.com/nnstreamer/TAOS-CI
+# @see      https://github.com/nnstreamer/nnstreamer
+# @author   Geunsik Lim <geunsik.lim@samsung.com>
+# @author   MyungJoo Ham <myungjoo.ham@samsung.com>
+#
+
+if [ -z $1 ]; then
+  echo "::error The argument (file path) is not given."
+  exit 1
+fi
+
+files=$1
+failed=0
+
+if [ ! -f $files ]; then
+  echo "::error The file $files does not exists."
+  exit 1
+fi
+
+echo "::group::Newline check started"
+
+num=0
+report=$(mktemp)
+finalreport=$(mktemp)
+for file in `cat $files`; do
+  newline_count=0
+
+  # If a file is "ASCII text" type, let's check a newline rule.
+  if [[ `file $current_file | grep "ASCII text" | wc -l` -gt 0 ]]; then
```

Comment:
Should be `$file` instead of `$current_file`

Reply:
applied!